sdk/dotnet/Pulumi/PublicAPI.Shipped.txt

```diff
@@ -23,6 +23,7 @@ Pulumi.AssetArchive.AssetArchive(System.Collections.Generic.IDictionary<string,
 Pulumi.AssetOrArchive
 Pulumi.ComponentResource
 Pulumi.ComponentResource.ComponentResource(string type, string name, Pulumi.ComponentResourceOptions options = null) -> void
+Pulumi.ComponentResource.ComponentResource(string type, string name, Pulumi.ResourceArgs args, Pulumi.ComponentResourceOptions options = null, bool remote = false) -> void
```

Comment:
I believe this is the only public API change that is illustrated in the example. Could you comment on why other changes are needed to be public?

Reply:
I've updated the PR so that this is the only public API change.